elements/pfe-switch/src/pfe-switch.scss

```diff
@@ -0,0 +1,164 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: switch;
+
+// Setting all toggle sizing vars
+// will be converted to em in styles so the graphic elments scale with type
+$font-size:             pfe-var(font-size);
+$toggle-width:          42px;
+$toggle-knob-diameter:  14px;
+$toggle-knob-offset:    5px;
+
+// Variables based on PF Next Switch
+$variables: (
+  toggle-knob: (
+    BackgroundColor: pfe-color(surface--lightest),
+    BackgroundColor--disabled: pfe-color(ui-disabled),
+    BoxShadow: '0 0.0625rem 0.0625rem 0rem rgba(3, 3, 3, 0.05), 0 0.25rem 0.5rem 0.25rem rgba(3, 3, 3, 0.06)',
+  ),
+  toggle: (
+    BackgroundColor: #d2d2d2, // @todo - didn't find a great color to match PF4
```

Comment:
@castastrophe couldn't find a color that made sense for this, any thoughts?

Reply:
I like your todo, we can look into if we're missing a color variable from the map or if we need to update the colors in the map currently at a later date. 👍